solr/modules/monitor/build.gradle

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'java-library'
+
+description = 'Apache Solr Monitor'
```

Comment:
That is so puzzling to anyone who isn't intimately familiar with Lucene Monitor.  I don't even think we should be calling this "Solr Monitor"; looks like infrastructure monitoring thing.  Possibly "Solr-Lucene-Monitor" but still... a puzzling name.

Reply:
You're right, if anything this _might_ be a part of some larger alerting system, but "saved search" is more accurate.

Reply:
Saved searches is a pretty indicative name. Percolator is also a known name for this kind of functionally.

Reply:
Interesting, I thought ES invented "percolator" as more of a metaphor... I wasn't aware that this is a more generic name. I was worried that "percolator" might clash too much with ES.